src/search/form/SearchInput.tsx

```diff
@@ -50,6 +47,7 @@ const InputText = styled.input`
 
   @media ${Device.mobile} {
     max-width: none;
+    width: 70%;
```

Comment:
mam wrażenie, że to jest ryzykowne i może się rozjechać coś... może lepiej ustalić marginesy?